libbeat/docs/command-reference.asciidoc

```diff
@@ -834,6 +834,11 @@ Sets up ingest pipelines for configured filesets. {beatname_uc} looks for
 enabled modules in the +{beatname_lc}.yml+ file. If you used the
 <<modules-command,`modules`>> command to enable modules in the `modules.d`
 directory, also specify the `--modules` flag.
+
+*`--enable-all-filesets`*::
+Enables all modules and filesets.  This is useful with `--pipelines` if
+you want to load all ingest pipelines.
```

Comment:
```suggestion
Enables all modules and filesets. This is useful with `--pipelines` if
you want to load all ingest pipelines.
```
Removing a bonus space.

Let's also explain the specific use case `--enable-all-filesets` addresses. For example to avoid having to do what is described in https://github.com/elastic/beats/issues/30916#issuecomment-1077657691 for every module manually.

Reply:
updated